app/assets/src/components/views/report/SampleDetailsSidebar/SampleDetailsSidebar.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { keyBy, mapValues } from "lodash";
+import PropTypes from "../../../utils/propTypes";
+import Sidebar from "../../../ui/containers/Sidebar";
+import RemoveIcon from "../../../ui/icons/RemoveIcon";
+import {
+  getSampleMetadata,
+  saveSampleMetadata,
+  getMetadataTypes
+} from "../../../../api";
```

Comment:
Should we add `path.resolve(__dirname, "app/assets/src/")` to solve this? 
Not sure how many conflicts we would have though...

Reply:
https://webpack.js.org/configuration/resolve/#resolve-modules